### ISSUE TITLE
Feature request: request history list

### ISSUE THREAD

**Issue by zjtv** [chapar-rest/chapar#46]
please add "the request history list" likes Postman. thx

**Comment by mirzakhany** [comment]
Good point, @zjtv I'll look into it. what you think about having it like a tab in treeview? so user can switch between the tree view and list of recently called apis.

**Comment by izturn** [comment]
good idea

**Comment by zjtv** [comment]
good idea
